### ISSUE TITLE
false positive Mathjax.document warning

### ISSUE THREAD

**Issue by zsviczian** [obsidianmd/eslint-plugin#162]
<img width="565" height="123" alt="Image" src="https://github.com/user-attachments/assets/fe55110a-ea25-43fa-9c03-b52932a403ef" />

```ts
let html: ReturnType<typeof mathjax.document> | null = null;
```

https://github.com/zsviczian/obsidian-excalidraw-extras/blob/a6d7447d714c1225c997cf215addafb27f77c71f/src/MathjaxToSVG/index.ts#L23

**Comment by cybaea** [comment]
For the maintainers: this is similar to #150 .
